tests/performance/common/common.go

```diff
@@ -283,10 +300,12 @@ func SendHttpRequest(method, reqApi string, body io.Reader) (error, *http.Respon
 	req.Header.Set("Content-Type", "application/json")
 	t := time.Now()
 	resp, err = client.Do(req)
-	utils.Info("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
 	if err != nil {
 		utils.Failf("HTTP request is failed :%v", err)
 		return err, resp
 	}
+	if resp != nil {
+		utils.Info("%s %s %v in %v", req.Method, req.URL, resp.Status, time.Now().Sub(t))
```

Comment:
can we improve this message ?

Reply:
Thank @rohitsardesai83 your suggestion is?

Reply:
`GET http://172.16.0.18:8080/api/v1/nodes/perf-node-h33cjb8q9v 200 OK in 1.345807ms`

Currently it is like the above message:)